packages/wrangler/src/mtls-certificate/cli.ts

```diff
@@ -0,0 +1,168 @@
+): Argv<UploadMTlsCertificateArgs> {
+	return yargs
+		.option("cert", {
+			describe:
+				"A certificate file (.pem) containing a chain of certificates to upload",
+			type: "string",
+			demandOption: true,
+		})
+		.option("key", {
+			describe: "A file containing the private key for your leaf certificate",
+			type: "string",
+			demandOption: true,
+		})
+		.option("name", {
+			describe: "The name for the certificate",
+			type: "string",
+		});
+}
+
+const uploadMTlsCertificateHandler = withConfig<UploadMTlsCertificateArgs>(
+	async ({ config, cert, key, name }) => {
+		const accountId = await requireAuth(config);
+		logger.log(
+			name
+				? `Uploading mTLS Certificate ${name}...`
+				: `Uploading mTLS Certificate...`
+		);
+		const certResponse = await uploadMTlsCertificateFromFs(accountId, {
+			certificateChainFilename: cert,
+			privateKeyFilename: key,
+			name: name,
+		});
+		const expiresOn = new Date(certResponse.expires_on).toLocaleDateString();
+		logger.log(
+			name
+				? `Success! Uploaded mTLS Certificate ${name}`
+				: `Success! Uploaded mTLS Certificate`
+		);
+		logger.log(`ID: ${certResponse.id}`);
+		logger.log(`Issuer: ${certResponse.issuer}`);
+		logger.log(`Expires on ${expiresOn}`);
+	}
+);
+
+export const listMTlsCertificatesHandler = withConfig(async ({ config }) => {
+	const accountId = await requireAuth(config);
+	const certificates = await listMTlsCertificates(accountId, {});
+	for (const certificate of certificates) {
+		logger.log(`ID: ${certificate.id}`);
+		if (certificate.name) {
+			logger.log(`Name: ${certificate.name}`);
+		}
+		logger.log(`Issuer: ${certificate.issuer}`);
+		logger.log(
+			`Created on: ${new Date(certificate.uploaded_on).toLocaleDateString()}`
+		);
+		logger.log(
+			`Expires on: ${new Date(certificate.expires_on).toLocaleDateString()}`
+		);
+		logger.log("\n");
+	}
+});
+
+function deleteMTlsCertificateOptions(
+	yargs: Argv
+): Argv<DeleteMTlsCertificateArgs> {
+	return yargs
+		.option("id", {
+			describe: "The id of the mTLS certificate to delete",
+			type: "string",
+		})
+		.option("name", {
+			describe: "The name of the mTLS certificate record to delete",
+			type: "string",
+		});
+}
+
+export const deleteMTlsCertificateHandler =
+	withConfig<DeleteMTlsCertificateArgs>(async ({ config, id, name }) => {
+		const accountId = await requireAuth(config);
+		if (id && name) {
+			return logger.error(`Error: can't provide both --id and --name.`);
+		} else if (!id && !name) {
+			return logger.error(`Error: must provide --id or --name.`);
+		}
+		let certificate: MTlsCertificateResponse;
+		if (id) {
+			certificate = await getMTlsCertificate(accountId, id);
+		} else {
+			certificate = await getMTlsCertificateByName(accountId, name as string);
+		}
+
+		const response = await confirm(
+			certificate.name
+				? `Are you sure you want to delete certificate ${certificate.id} (${certificate.name})?`
+				: `Are you sure you want to delete certificate ${certificate.id}?`
```

Comment:
confirmation path for `id` & `name` are hit but seems like the ternary isn't triggered for just the `id` path, the logger below that is triggered by the 2nd return is also not covered. Please provide a test scenario for this too.